compiler/packages/react-compiler-runtime/package.json

```
@@ -9,7 +9,7 @@
     "src"
   ],
   "peerDependencies": {
-    "react": "^18.2.0 || ^19.0.0"
+    "react": "^17.0.0 || ^18.2.0 || ^19.0.0"
```

Would you mind relaxing the 18 version range too? Thanks!

```suggestion
    "react": "^17.0.0 || ^18.0.0 || ^19.0.0"
```